### ISSUE TITLE
Test duration is always zero

### ISSUE THREAD

**Issue by kugel-** [Snaipe/Criterion#191]
On a self-compiled 2.3.1, I constantly get reported test duration of 0,00s, regardless of how long the test takes (4 seconds in the below example). I remember seeing this in 2.3.0 as well. The 2.2.x series works fine, though.

```
$ time env LANG=C ./test_all --filter "cexec/cexectimeout1" --verbose --ascii
[----] Criterion v2.3.1
[====] Running 3 tests from cexec:
[SKIP] cexec::cexecexit42: Test is disabled
[RUN ] cexec::cexectimeout1
[PASS] cexec::cexectimeout1: (0.00s)
[SKIP] cexec::cexectimeout2: Test is disabled
[====] Synthesis: Tested: 1 | Passing: 1 | Failing: 0 | Crashing: 0 

real	0m4.017s
user	0m0.004s
sys	0m0.004s

```

**Comment by Snaipe** [comment]
This got fixed after the release in e23699f459e, and should be shipped with the upcoming v2.3.2.

**Comment by kugel-** [comment]
Counts to me as fixed
